src/Databricks/custom/Update-AzDatabricksWorkspace.ps1

```diff
@@ -184,6 +184,11 @@ function Update-AzDatabricksWorkspace {
             }
             if ($hasEncryptionKeyVersion) {
                 $workspace.EncryptionKeyVersion = $EncryptionKeyVersion
+            } else {
+                if ($workspace.PrepareEncryption)
```

Comment:
Is it because that you cannot set `EncryptionKeyVersion` unless the workspace has prepared for encryption?

Reply:
Yes